ckanext/datagovtheme/templates/snippets/gtm/gtm_datalayer.html

```diff
@@ -11,14 +11,14 @@
 
       <script type="application/ld+json" class="jsonld-dataset">
         dataLayer = [{
```

Comment:
do we need to push this object TO anything?
is `datalayer` created already or do we need to instantiate the var?

Reply:
This would be the next thing I try if this doesnt work. Made comment here: https://github.com/GSA/data.gov/issues/4783#issuecomment-2214129657

Reply:
if `window.dataLayer` exists already this would replace it with this. Typically, you'd push so as not to wipe/replace anything pre-existing.

Reply:
That's a good point. I don't think that the `dataLayer` exists prior, but I'm not sure how to test that. Definitely very open to advice/contributions for the front end stuff.